app/models/post.rb

```diff
@@ -1,4 +1,9 @@
 class Post < ApplicationRecord
+
+  def self.reverse_order
+    Post.all.reverse
```

Comment:
I wonder whether you need `Post` here. I'd be interesting to know the scope. I have a suspicion you could call `all.reverse`